config/compareWarpAssembleCoadd.py

```diff
@@ -0,0 +1,16 @@
+import os.path
+
+# Load configs from base assembleCoadd
+config.load(os.path.join(os.path.dirname(__file__), "assembleCoadd.py"))
+
+# 200 rows (since patch width is typically < 10k pixels
+# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
+config.assembleStaticSkyModel.subregionSize = (10000, 200)
+config.assembleStaticSkyModel.doApplyExternalPhotoCalib = True
+config.assembleStaticSkyModel.externalPhotoCalibName = 'jointcal'
+config.assembleStaticSkyModel.doApplyExternalSkyWcs = True
+config.doFilterMorphological = True
```

Comment:
As in `coaddBase.py`, these configs are DRP specific and not always desired for AP template building. I'd prefer to have them set in `DRP.yaml` instead of for all DECam things.

Reply:
See above - this file no longer exists.